### DISCUSSION TITLE
update how to

### DISCUSSION THREAD

**Discussion by montyfert** [paukstelis/octoprint_deploy#125] [Q&A] [answered]
hi do i update via octoprint or by sudo 
octoprint_deploy/octoprint_deploy.sh  
7 update
thanks
martin

**Comment by paukstelis** [comment, marked as answer]
To update OctoPrint, you do it through OctoPrint UI. octoprint_deploy update is only to update octoprint_deploy.

**Reply by montyfert** [reply]
thanks 
and thanks for the octoprint_deploy its great and easier to use.
martin
